### ISSUE TITLE
Shortcut to open Cmder in a chosen folder

### ISSUE THREAD

**Issue by SudatiSimone** [cmderdev/cmder#2428]
Hi,
I follow the instructions in "Shortcut to open Cmder in a chosen folder".
Now I could open the Cmder from any folder and It works. 
The problem is that the terminal's startup folder is fixed to 
" C:\Users\susim\Desktop\folder  " and it doesn't change in response to the position
where I run Cmder.
( right-click -> cmder here)

Thanks

**Comment by daxgames** [comment]
There is nothing in Cmder that 'fixes' it to the folder you named.

What version of Cmder are you using?

What is the text of the Conemu Task that is being launched?

**Comment by SudatiSimone** [comment]
I found the problem but I don't know how to solve it.
I set the default startup directory following these instructions:
 Open Cmder -> Settings -> Tasks -> select {cmd::Cmder} (in my case). On the bottom (below textarea) you have button named "Startup dir...", and than select startup directory.
In this way when I use right-click & cmder here, the directory is fixed to my choice.
I don't want to change my startup directory choice because it is comfortable.
Is there any solution?
